### ISSUE TITLE
Alarm trigger time changes with the time zone

### ISSUE THREAD

**Issue by djdarkstep** [ataradov/alarm-me#7]
Alarm time would change when travelling to another time zone and the alarms are going to trigger with a time zone offset.

**Comment by ataradov** [comment]
Yes, I know. I doubt I will ever fix this. I really hate Android development and it works for me as is.

**Comment by djdarkstep** [comment]
I'm using your code as inspiration and a base for a project of mine. Do you happen do have the slightest idea for a workaround?

**Comment by ataradov** [comment]
Not at at all. I powered through this stuff when development of another application I was using for this stopped. But I really have no idea, I relied on google and stackoverflow a lot to do that.
